arrow/src/main/java/org/apache/iceberg/arrow/vectorized/ColumnarBatch.java

```diff
@@ -36,17 +36,17 @@
   ColumnarBatch(int numRows, ColumnVector[] columns) {
     for (int i = 0; i < columns.length; i++) {
       int columnValueCount = columns[i].getFieldVector().getValueCount();
-      Preconditions.checkArgument(numRows == columnValueCount,
+      Preconditions.checkArgument(
+          numRows == columnValueCount,
```

Comment:
I don't have a problem with including the condition on the previous line. Is the rule that if one argument starts with a newline that all arguments must be?

Reply:
as mentioned in the PR description I was just applying whatever formatting rules are defined in `intellij-java-palantir-style.xml`.